### DISCUSSION TITLE
old company email address getting forwarded commits from new company how do you stop this?

### DISCUSSION THREAD

**Discussion by smbruck** [community/community#56228] [New to GitHub] [answered]
My old startup email, which I no longer have access to, is being forwarded git commits from a company I am contracting with. How can I stop this from happening?  I can find no place where this email is in github.  Thanks.

**Comment by airtower-luna** [comment]
What exactly do you mean by "being forwarded git commits"?

**Reply by smbruck** [reply]
what I see in my email (gmail) is
from: my personal email address associated with github
to : my old work email
cc: my personal email (same as above)
subject is always : FWD [new company]  what was done
then in email body
---forwarded message---
message header with 
subject, date from,  etc , all from new company

The question is where is git storing my old company email and how do I stop the forwarding

thanks again for helping

**Reply by smbruck** [reply, marked as answer]
OK, i feel like an idiot, it was an automatic forward in my thunderbird email client. I will close this issue

**Reply by 2bbac** [reply]
Adresa mea de Yahoo este folosita abuziv de componiendo unde am facut investitia [Thunderbird](url) si pe mine mau blocat au schimbat numele meu si adresa mea ce pot sa fac

**Reply by pablitomilenio** [reply]
I have the same problem

I dont seem to have my old employers email anywhere on github anymore but they are getting replies on repositories they have never interacted with

**Comment by kutorodan** [comment]
My. Name is. Naoko. Kuroda. Hyougo. From. 
osaka. In. Lave. 6th. Year.  To. Tokyo. 1th. Year. To. Indonesia. Rast. Year. 7manth.  Ago. Stay.  Please. Exuse. My .Barseday.  1966. 8.1.    Tokyo. Shibuya. Ebisu. Global. Shitp. Saratry.  Wark. At. Cecuruty.
